lib/carrierwave/base64/orm/mongoid.rb

```diff
@@ -0,0 +1,23 @@
+require 'mongoid'
+
+module Carrierwave
+  module Base64
+    module Mongoid
+
+      def mount_base64_uploader(attribute, uploader_class, options = {})
```

Comment:
this class duplicates the existing adapter for activerecord. We should either make one base class, and inherit from it, or just use one class for both adapters